entrypoint.sh

```diff
@@ -38,8 +51,12 @@ download_threescale_config() {
   unzip nginx.zip
   # Most docker PaaS doesn't allow docker to run as root
   # lets use the 8080 port instead of 80
+  if [ $STDOUT_WRITABLE = "true" ]; then
+    sed -E -i "/server\s+\{/a access_log /dev/stdout combined;" nginx_*.conf
```

Comment:
By default there's no ```access_log``` directive... 

Ok I will change the code to do the following:

If /dev/stdout is writable  -> symlink log/access.log to /dev/stdout
If /dev/stdout is no writable -> symlink log/access.log to /dev/null
And always add the access_log directive to the log pointing to log/access.log 

This should be a more "consistent" solution.